11-if/02-if-statement/05-challenge-userpass/01-1st-challenge/01-challenge/main.go

```diff
@@ -34,4 +30,20 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	l := len(os.Args) - 1
```

Comment:
I think we should not use small letter L variable as it can be confused as number one

Reply:
Fixed "l" to "lenOs"